src/EditorFeatures/CSharpTest/CommentSelection/CSharpToggleLineCommentCommandHandlerTests.cs

```diff
@@ -106,7 +106,7 @@ class C
 {
     void M()
     {
-[|        //var i = 1;|]
+        [|//var i = 1;|]
```

Comment:
This has moved the caret. Suggest leaving the caret unchanged (as per spec).
Caret should be hard left against "var".